mypy/private/mypy.bzl

```diff
@@ -80,10 +80,14 @@ def _mypy_impl(target, ctx):
         # file roots?
 
     unique_generated_dirs = generated_dirs.keys()
+    generated_custom_imports = []
+    for generated_dir in unique_generated_dirs:
+        for custom_import in custom_imports:
+            generated_custom_imports.append("{}/{}".format(generated_dir, custom_import))
```

Comment:
I guess some of these may just not exist?

Reply:
Yea. I figured that shouldn't hurt

Reply:
happy to roll with it -- only a mild concern about potential size of the env var, but that's a problem for another time